requirements.txt

```diff
@@ -1,6 +1,6 @@
 # runtime
 psutil>=5.6.3
 enum34;python_version<"3.4"
-PySide2==5.14.1
+PySide2>=5.14.1
```

Comment:
PySide2 5.15 and up still causes lots of errors on Windoze so IMO it should be `PySide2==5.14.2.3`.

Reply:
5.14.2 has issues too (drag&drop could freeze the app).
So we need to keep 5.14.1 until we move to qt 6.

Reply:
>  So we need to keep 5.14.1 until we move to qt 6.

It's impossible to fix macOS Big Sur then, because 5.14.1 is not available for it.

Reply:
AFAIK 5.14.1 is not available for Python 3.9. It is not specific to macos.

So we can make it optional to the python version (even if it can create some known issues).